.github/problem-matchers/kotlin.json

```diff
@@ -0,0 +1,40 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "kotlin",
+            "severity": "error",
+            "pattern": [
+                {
+                    "regexp": "^e\\:\\s(?:(?:(.*):(?:\\s\\()?(\\d+)(?:(?:\\,\\s)|\\:)(\\d+))(?:\\))?\\:\\s)?(.*)$",
```

Comment:
I came up with them. Because we use two kotlin versions in our project (buildSrc, and regular project) we have two kinds of logs for kotlin compilation errors.